### DISCUSSION TITLE
As a very excellent plugin, whether to consider multiple languages

### DISCUSSION THREAD

**Discussion by laterdayi** [rydmike/flex_color_scheme#203] [Q&A] [answered]
As an excellent plugin, should we consider providing multi-language versions of the documentation and theme playground, such as Chinese? I believe this will make everyone more aware of flexcolorscheme and contribute to it

**Comment by rydmike** [comment, marked as answer]
Multiple language support would certainly be nice. It is certainly something I would like to have long term wise, but I cannot commit time to it at the moment.

For the doc site https://docs.flexcolorscheme.com/ that uses https://docs.page/ it can in theory be added, but of course somebody would have to be committed to translating it (or review machine translations) and updating the translated pages as well whenever the english original changes, which is quite often.

The doc site is intended only for developers, but yes I do realize not all developers speak English fluently, even if they probably have to read dev docs in English a lot, so it would certainly be nice to support other languages.

As for the **Themes Playground** companion app. The current version was originally only a small demo of what you can do with **FlexColorScheme** and intended as a tutorial step on how to do it. Since it was not planned to become so big and such a popular developer Fluttering theming tool. Since it was only a tutorial step, it left out a lot of things on purpose that you would add from the beginning, if you aim for a large app. One of those things is Flutter localizations. 

Adding it now later, while possible, would be a big task. So I'm not planning on adding localization support to the current version. I am however planning a totally new and much improved version of the Themes Playground (code name "V10" for now :smile:). That version will certainly include localization from the start. Providing label translations to that version, to any language, can then be crowd source to its users, and of course use machine translated labels as a starting point.

**Comment by laterdayi** [comment]
Thank you for your contribution and look forward to it
